server.js

```diff
@@ -1,5 +1,21 @@
-const app = require('./app')
+const mongoose = require('mongoose');
 
-app.listen(3000, () => {
-  console.log("Server running. Use our API on port: 3000")
-})
+const DB_HOST =
+  'mongodb+srv://Serhii_Mukhin:Ir3jTppMXV8FB93M@cluster0.nldskyi.mongodb.net/db-contacts?retryWrites=true&w=majority';
```

Comment:
```suggestion
const DB_HOST =
  'mongodb+srv://Serhii_Mukhin:Ir3jTppMXV8FB93M@cluster0.nldskyi.mongodb.net/db-contacts?retryWrites=true&w=majority';
``` PORT та DB_HOST винесіть у змінні серидовища